example/basic_example/main.go

```diff
@@ -0,0 +1,42 @@
+// ADAPTED FROM OPENTELEMETRY BASIC EXAMPLE
+
+package main
+
+import (
+	"context"
+
+	"github.com/honeycombio/opentelemetry-exporter-go/honeycomb"
+	apitrace "go.opentelemetry.io/api/trace"
+	"go.opentelemetry.io/sdk/trace"
+)
+
+func main() {
+	trace.Register()
+	ctx := context.Background()
+
+	// Register the Honeycomb exporter to be able to retrieve
+	// the collected spans.
+	exporter := honeycomb.NewExporter("API_KEY", "DATASET_NAME")
```

Comment:
make this a flag (or at least a `var`), so that people can just run the example without needing to change the source code.